crates/red_knot_ide/src/completion.rs

```diff
@@ -0,0 +1,43 @@
+use ruff_db::files::File;
+use ruff_db::parsed::parsed_module;
+use ruff_python_ast::visitor::source_order::{SourceOrderVisitor, TraversalSignal};
+use ruff_python_ast::AnyNodeRef;
+use ruff_text_size::TextSize;
+
+use crate::Db;
+
+pub struct Completion {
+    pub label: String,
+}
```

Comment:
This could, probably return a `&'db str` but I'm fine with a `String`.

Reply:
Yeah it eventually has to get turned into a `String` anyway. And I figured this would all be refactored soon anyway.